test/MediatR.Tests/PipelineTests.cs

```diff
@@ -1,14 +1,12 @@
+using System.Collections.Generic;
 using System.Threading;
+using System.Threading.Tasks;
+using Shouldly;
+using StructureMap;
+using Xunit;
 
 namespace MediatR.Tests
 {
-    using System;
-    using System.Collections.Generic;
-    using System.Threading.Tasks;
-    using Shouldly;
-    using StructureMap;
-    using Xunit;
-
```

Comment:
I understand removing unused references, but I don't know the implications of moving usings from inside the namespace to outside, any reason for doing so?

Reply:
There is no benefit to using a `using` statement inside or outside of the namespace. I think the only difference is the scope. So the `using`s inside the namespace are only accessible inside the namespace and the `using`s outside are for the whole file. Since there is only one namespace in this file, it won't matter.

When I saw this in the file, my neurotic nature wanted to put them together as they belong together.

Reply:
If you want, I can go over the code and put them all together and order them alphabetically.

Reply:
If they are functionally the same it doesn't matter to me, I'll defer to Jimmy.  I've always put them outside but have seen them inside namespaces as well.

Reply:
Ok, I'll leave it up to @jbogard to decide if I roll this back, if I format and order the rest of the `using`s in the codebase or if this is ok.

Reply:
I prefer consistency tbqh

Reply:
Would you like me to roll it back or would you like me to look through the codebase and format the other places?

Reply:
If it's not consistent in the codebase today, leave it alone.

Reply:
Should be rolled back now.